fe/user/src/pages/InquiryPage/Editor/index.tsx

```diff
@@ -16,6 +17,8 @@ type Props = {
 };
 
 export const InquiryEditor: React.FC<Props> = ({ currentCategory }) => {
+  const { isMobile } = useDeviceTypeStore().deviceType;
```

Comment:
```suggestion
  const { isMobile } = useDeviceTypeStore((state) => state.deviceType);
```
으로 select 해서 가져와야 하는 것으로 알고 있어요. 지금은 setDeviceType 밖에 없지만 다른 값들이 들어오는 경우 지금처럼 작성되어 있으면 `useDeviceTypeStore` 모든 값을 참조하게 될거에요

Reply:
테스트할때 해놓은 값을 변경 안하고 다 복붙해놨었네요😅 수정 했습니다

Reply:
굉장히 빠른데요?